keras_cv/models/object_detection/predict_utils.py

```diff
@@ -15,12 +15,13 @@
 import tensorflow as tf
 
 try:
-    from keras.src.engine.training import _minimum_control_deps
-    from keras.src.engine.training import reduce_per_replica
+    # To-do: these imports need to fixed - Issue 2134
```

Comment:
I would just toss the whole link on a new line for easy browsing
`# https://github.com/keras-team/keras-cv/issues/2134`

Reply:
done